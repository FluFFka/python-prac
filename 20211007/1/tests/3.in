print(Pareto((87743, 61327), (23989, -93813), (21440, -88577), (-85491, 41947), (-77294, 17698), (-30702, -43902), (-65709, 78113), (74123, 90800), (82641, -25478), (-86501, -53220), (25131, -63970), (-57431, -64688), (-1748, -57023), (-55872, -10763), (95779, -708), (-43904, 53569), (-23875, 15283), (-27099, -58580), (-47775, -68506), (46039, -64493), (-56350, -19205), (-29151, -17061), (-86639, 27626), (79040, -30164), (-7864, 94754), (-81143, -28930), (-67429, 80869), (9506, -97307), (-88842, 26795), (-72444, -78629), (-69928, 2379), (38055, -81749), (41948, 47368), (-56077, 25652), (52195, -82032), (46521, -28946), (-28385, -59565), (48285, -26070), (-96264, -66041), (-21843, 48053), (47368, -88199), (54292, 97887), (5431, -35894), (-2198, -59433), (-89227, 84738), (-7000, -71960), (-14205, 56733), (17668, 69394), (-53471, -85444), (42891, 96781), (-62145, 9986), (-994, 6140), (-89137, 8578), (7788, 78539), (73101, -76192), (68036, -8647), (-32092, 14745), (-71839, -67373), (82279, -75724), (-57360, -21076), (31034, 3833), (-35177, 10728), (87545, -56655), (-23795, -57841), (78755, -89336), (-42804, 48736), (19494, -30552), (-48881, -98362), (29995, 85319), (-54150, 96457), (-18926, -3767), (15011, -91818), (-82578, -37873), (-33563, -1694), (83093, 13631), (-27998, 62828), (-56019, 63437), (90605, -94003), (82093, -10229), (-39315, 45814), (93871, -4717), (-58476, -27846), (23247, 60755), (56342, -95505), (23548, -89159), (-72752, 44190), (50861, -97256), (21217, -66420), (27134, -25602), (49046, -8780), (49353, -2702), (-20241, -66267), (99491, 92432), (-54924, -20562), (-87464, -13248), (65505, -55197), (-90736, 28129), (22328, 90698), (-95270, -51296), (39714, -57637), (-339, -84644), (94781, -32548), (30834, -86837), (87866, -27755), (-99388, -22660), (84605, -16282), (91498, -64988), (95726, 22250), (-67650, -46839), (97163, 26374), (47121, -76635), (-12353, -40464), (-81966, 47718), (-51137, 87459), (45023, -25246), (-64582, -8514), (-7324, -55219), (-12687, 71346), (-82204, -16688), (3583, 57921), (75758, 85305), (71545, 76831), (20054, -69744), (-73375, 80937), (31204, -37596), (-48732, 33247), (-48970, 58858), (4213, 37735), (73050, 70533), (59542, -4033), (-18036, 25166), (54883, 88388), (2507, 53919), (56551, -67615), (35093, 11609), (69124, 45902), (59758, -72616), (33402, 82201), (-26121, -2847), (-40827, -27521), (-49582, 57154), (4151, 88961), (-48029, 94953), (62698, 55477), (-41000, 48201), (522, -44817), (-31447, -26840), (59490, 38196), (30284, 18121), (54444, -24787), (-54984, -44227), (-42846, -35552), (-25457, -50108), (62232, -71024), (4880, 24797), (66215, 89519), (-21444, -41008), (-2258, -2978), (22859, -14773), (23077, 85593), (-31634, -58399), (-50259, 99311), (62235, 69094), (64210, -20429), (-63336, -69550), (74317, 73788), (96484, 87335), (79414, -49184), (80108, -26951), (82692, 66621), (2685, -93148), (-63327, -54900), (-3689, -15911), (-65942, -20709), (-16993, -7180), (-63838, -38236), (-46800, 38542), (-58548, 78525), (54836, 72163), (-71511, -13996), (56009, -7527), (-14274, 98592), (72987, -89682), (84333, 95615), (29139, -50898), (47613, 10768), (-76751, 60140), (-93497, 45835), (-33472, 44549), (-58291, 37085), (-89978, 92626), (75803, -9348), (-65254, 44914), (44114, -58565), (-22780, 16785), (-53916, -5609), (-80993, -4671), (-73724, -66923), (-96376, 2670), (65295, 7994), (35608, 52790), (-7748, 31751), (-84418, 57759), (-85329, -4832), (25852, -14870), (22047, 84921), (85557, -96992), (-36791, -29030), (-76314, -90463), (-81762, -6491), (-24780, -94643), (-77845, 23422), (44818, -47015), (83692, -88520), (-22097, -87594), (52187, 5261), (-79076, -8214), (-31144, 94485), (-77750, 32875), (78666, -90034), (-43293, 26210), (-60832, 96351), (46083, 5341), (-30257, -77837), (64749, 10513), (71299, -39426), (-15624, 16966), (-42811, 27), (-19561, -30959), (89911, -37753), (12871, -97867), (-20323, -95610), (39522, -44341), (-40921, -28445), (-3100, -59720), (-80824, 3276), (-8454, -66970), (3819, -77553), (-45295, -73722), (27452, -85853), (-1214, -72171), (-45, -82913), (-68444, -18903), (8790, -90869), (86058, -67627), (83845, 99137), (91495, -38914), (75872, 42949), (-73289, 39584), (-30753, 91231), (-38259, 61723), (-56233, 60063), (-53268, -3223), (-71803, 56191), (25430, -4053), (65155, -3524), (-25386, -67009), (74060, -46205), (-57405, 68882), (3047, 4245), (9603, -57958), (-74155, 83307), (-51760, 95375), (-48939, -26666), (38910, -65277), (76971, 9105), (-78830, 22531), (-57361, 3230), (-10826, 50850), (-39055, -99640), (-68006, 49172), (44360, 2843), (-21406, -85736), (38499, -13680), (21134, 22507), (-23968, 75490), (-67183, -4047), (96661, -89575), (55928, -40833), (28950, -34975), (-66327, 47640), (89240, 85063), (-43063, 76145), (-93994, -77979), (83071, -25450), (-17589, -34301), (55634, 64427), (81001, -68909), (56796, 14421), (-51044, 35223), (57780, 1629), (32142, -78629), (-65279, 93815), (-77636, 26277), (4323, 85374), (65587, -9731), (43716, 80391), (8824, 46457), (39422, 13885), (-24575, -72815), (19905, 42425), (86002, -66707), (-63916, -58000), (-55901, 38017), (-37099, 93060), (-72352, -41143), (85014, -31652), (-96161, -69963), (-46346, 69161), (-34684, -61925), (20358, 97961), (-41518, 43293), (60039, 3557), (29427, 26332), (-15519, -79101), (-36340, -67584), (30781, 26245), (-70278, -58237), (-93638, -99039), (36626, 25633), (-25758, -98944), (48835, 79818), (39783, 13115), (-61619, -78450), (16343, -98821), (88035, -6624), (95893, -23228), (-31451, 2843), (38721, -11969), (89006, 82715), (-53776, 775), (86020, -49985), (-54156, -81039), (-6153, 80025), (56117, 51692), (-41387, 32866), (77521, -12601), (84497, 17736), (-34480, 73035), (11205, 47837), (-86441, -8814), (-61963, -44228), (68873, -72960), (37677, -84667), (31385, -78040), (65986, -65737), (39173, 18806), (74147, 36893), (8257, -35278), (66482, 39619), (-85283, -30179), (-92662, 97700), (50931, -80403), (-82549, 94256), (-2539, -14048), (92344, 65314), (-83970, -12497), (-22407, 47594), (-60889, -97236), (38570, -78449), (-10525, -32806), (-75734, -27096), (-8717, 91969), (-54079, -56034), (-74178, -5120), (-94082, -81257), (-29475, 77386), (84331, 97511), (27646, -39834), (42177, -74168), (23709, -5642), (-76248, 42529), (-43638, -45544), (-46877, -5403), (-88315, 1730), (-70639, 23952), (15813, 95918), (-80309, -58885), (-37027, -88168), (71291, -89722), (-56981, -11896), (-75900, -10407), (60727, -54125), (-34167, 81010), (13519, -4078), (68067, -94161), (-85912, -11530), (47837, -97689), (-39498, 35627), (8193, 58271), (51110, -63764), (81504, 12663), (-95472, 10490), (-97505, -21186), (77687, -23834), (1243, -53327), (20008, -22110), (-92626, -97099), (-44600, 7248), (33484, -82044), (-95215, -2036), (-30916, 24939), (7976, -29233), (-54864, -40943), (-19342, -95634), (-87464, 51204), (29840, -42222), (-66251, 45075), (-43714, 57432), (-97794, 22989), (-57880, -18366), (-97536, 20444), (-10873, -83525), (67366, 69191), (-89867, -91803), (-93889, -49897), (-39316, -12911), (-75857, 33366), (-59193, 96054), (3558, 72754), (73704, -60897), (-61880, 87004), (-81525, -58670), (-62622, 41436), (-19839, -71246), (-875, 30294), (-14734, 5455), (43736, -64138), (19136, 60612), (-93115, 572), (23199, -48330), (91662, 96679), (42237, -90876), (39235, -19051), (49678, 96524), (-33707, -61435), (82428, 86268), (-31894, 17631), (-20361, 46724), (30347, 82093), (27791, -13905), (27315, -38423), (-78973, -92334), (-35507, 20482), (98508, 59476), (-77374, 99009), (-22157, -70271), (54419, -12003), (70766, -37702), (-62183, 68223), (-8668, -9510), (-65211, 25334), (-34857, 33103), (17939, 27595), (-27395, -40668), (-99850, 25938), (-12172, -29405), (80520, -83442), (76732, 24618), (54782, 53613), (34186, -81347), (-67769, 239), (61961, -13418), (-12386, -59271), (-31385, 89560), (54829, 79521), (-92429, 79504), (87090, 66418), (85000, -10966), (-12291, -21455), (-99706, -82480), (-9613, 92778), (66118, -65247), (-54661, 26155), (-88158, -14086), (63293, -64649), (54555, 52200), (-1633, 15146), (-43880, 93112), (61718, -79392), (54332, 19411), (26970, -20461), (86281, -72446), (-60551, -55882), (-80579, 9059), (85507, -67103), (-45950, -16783), (86879, 14907), (-51474, -37094), (-89391, 15994), (50530, 59583), (-97975, 70186), (97417, -25504), (35480, -22799), (54967, 12172), (24144, -99260), (-41546, -89147), (90818, -94313), (-73656, 54859), (8876, 88754), (-46795, -47821), (75615, -77802), (73347, -76823), (34272, -7056), (-13794, 30405), (4063, -54019), (83667, -62056), (-32693, 38443), (29547, -53748), (-65404, -98654), (63009, -33816), (7003, -45087), (-35759, -2933), (-84560, -53126), (-85309, 1613), (-7875, -90348), (-77672, -74555), (-26277, -58441), (-76070, -38800), (14052, -4318), (-17553, -79651), (73545, -19586), (71537, 6007), (40231, 19608), (6101, -99928), (40770, 74615), (-52177, 79583), (85190, -19819), (96793, 13352), (60400, 48879), (83850, -97824), (-87412, -45543), (-11815, -45099), (94966, 25160), (-12476, 99754), (21817, -86750), (11706, 85875), (-27326, 4858), (-85698, 96176), (40147, -30727), (-82813, 5280), (-41954, -15101), (99625, -89468), (71966, -20215), (75578, 31312), (72324, -48728), (83600, 38494), (70633, -26678), (95253, -46822), (-28488, -38412), (-82647, -93233), (73425, -28006), (-41147, 59299), (68663, 61870), (79415, -24108), (-97496, 75699), (83826, 42522), (44753, 28234), (70793, 25097), (72957, 52186), (54960, 78408), (90839, -40148), (24603, -77152), (-65124, 16463), (-19970, -88138), (28974, 23590), (14468, -36332), (12706, 71519), (37168, 45410), (-30750, -45443), (13434, 45671), (-89727, 25071), (40857, -85030), (-8843, -15975), (89401, -94924), (7062, 74924), (-16048, 69306), (-48303, 28131), (39055, -70782), (30930, -88353), (-4108, 14522), (65041, -38987), (83475, -27330), (58375, -55076), (-30041, -81210), (28536, -7748), (-17718, 64682), (34196, 15980), (-344, -27602), (-26311, -34655), (64064, -10776), (-67842, 93636), (-974, 74864), (-74780, 28094), (81638, -299), (81530, -24983), (-38292, 72854), (-89465, -55570), (85480, 81202), (31470, -48622), (54719, -18720), (-47073, 43010), (-66446, -11553), (58418, 89999), (-76560, -9619), (56367, 1294), (21143, 75107), (-47162, 16695), (67152, -52905), (46781, -4138), (-7872, -82730), (-77876, 51387), (-70375, -73044), (-14317, -98982), (70465, 9147), (51669, 69043), (17422, -92072), (11463, 67749), (-93744, -22311), (27844, 14095), (-37988, -26573), (40059, 29366), (68602, -80432), (-51420, -26887), (20404, 5952), (65123, 17210), (42672, 23301), (-75046, 11699), (-80915, -83825), (36894, -62843), (-19966, -13292), (17415, -36483), (63397, -49275), (58350, -74917), (99161, 82650), (-34029, 56711), (23053, -10937), (-40774, 6734), (-76318, 60736), (6358, 90745), (-74562, -66251), (-26770, -32907), (33459, 74076), (87458, -15959), (68083, 20317), (17053, -2201), (54533, -61345), (-95794, 26884), (-66696, 27849), (33089, 31783), (-34024, -17096), (-64958, -49320), (-31149, 49062), (63295, -48204), (-93597, 23130), (-51223, 14093), (18057, -36470), (-80105, -87044), (-49156, 6369), (40901, -66953), (93070, 94327), (16439, 90871), (91831, 95231), (53189, -94914), (-69623, -61532), (95247, -6389), (18684, 54913), (3503, 31893), (-21441, -69651), (93314, 96841), (-6038, 14143), (-40706, 92705), (-12442, 15091), (-3394, -60861), (76738, 54319), (-99565, 73210), (32341, 35739), (20006, 47575), (51520, -96839), (56150, -61501), (-44531, -9235), (96198, -9991), (63289, -43194), (4113, 23877), (-21107, -2259), (-69750, 20501), (-41426, -40318), (-25181, -42565), (41880, -94900), (9712, -13969), (-47209, 65963), (-38178, 98048), (-54954, -54495), (38859, -54281), (-2667, 65446), (11781, 89320), (-50254, -51684), (82257, 16524), (14982, -51751), (31095, -63740), (-52300, -86587), (63479, -33042), (20979, 11698), (-47982, -11185), (-84748, 63909), (62696, -9243), (-46916, 31904), (33258, 58346), (-66605, -85718), (32460, -71420), (10324, 83975), (-17278, 40463), (20309, 39408), (63247, -14361), (-10552, 97377), (-87033, 22124), (-47252, -96929), (-59617, 9345), (70763, -62040), (-39261, -93885), (-40713, 85938), (-93684, 96426), (-41483, -80827), (17944, -4218), (-31838, -17484), (48412, -52033), (57622, -33245), (84946, 51819), (-99098, -95430), (86183, 27802), (14862, -23331), (-81193, 25921), (-93222, -4709), (-74830, -60834), (95261, 57245), (-84735, -88376), (77709, 85137), (-43586, -95331), (13546, 58349), (-79541, -90973), (88331, 92864), (-52977, 67338), (16370, 34208), (-60350, 28260), (36439, -9301), (29441, 56238), (-97059, 99159), (-25066, 33809), (-63095, -40210), (49377, 16452), (-78387, 11556), (78980, 63569), (-92260, -57512), (19800, -14786), (31403, -68317), (24770, -20323), (70539, 76717), (59496, -59504), (59459, 29302), (6930, -26749), (31532, 37466), (-21433, 58677), (68367, -49398), (38124, 33121), (4600, 60864), (-71241, 36633), (-66471, 6798), (-88751, -69151), (-5716, 80494), (88349, 71880), (-81329, -12068), (-47402, 94611), (-91756, 88629), (-74396, 38549), (-68930, 84953), (-89273, 7692), (-37561, -92628), (-48899, 59218), (62901, 68988), (5034, -45832), (-74056, 38887), (-27860, 14338), (89134, -53745), (30374, -73446), (28993, 57976), (-36791, -28791), (-13903, 13096), (13515, -67215), (-34723, 50932), (-56804, -79345), (54600, 60446), (21136, 25557), (-76337, -60221), (-66618, 98022), (76842, -14353), (-29025, 67839), (17787, 19810), (39511, 44626), (97258, -75226), (-81792, -15876), (-75567, 48153), (8266, 91156), (-8308, 90870), (-75560, -39181), (84953, -99556), (93438, 19393), (26471, 83669), (-83161, 96225), (-90113, -31226), (-8803, 88297), (-4528, -98285), (39730, -40440), (31225, -95133), (43033, 45250), (45528, -81225), (-83525, 2209), (55513, -3542), (-28524, 26586), (-79441, -28259), (67049, 87680), (11988, -87165), (70575, 35387), (-38643, -68857), (-96061, 90967), (48737, -23379), (-19409, -89533), (-99682, 65127), (87604, -20959), (-44302, -25400), (30827, -52051), (-63814, -69704), (75005, 80003), (85917, 38991), (-55399, -7971), (57085, 43680), (-98464, -57346), (-11351, -82276), (71774, 33239), (-38953, 88292), (5520, -7000), (-72040, -80169), (-38468, -22585), (-89389, 14683), (89287, -5163), (-23069, -53100), (57388, -96836), (-63659, -97148), (56628, -50991), (-5047, 39173), (32413, 65722), (-19715, -67870), (71918, 40254), (65994, 83844), (-71976, -26608), (44620, -14771), (-30713, 30228), (56346, 60388), (79552, -24094), (-60904, -55448), (-66204, -25928), (7404, 50288), (73066, 50886), (45223, 37112), (76251, -81377), (35091, -1201), (-40891, 82163), (46046, -40751), (5496, 99299), (-62782, -93808), (-68275, -84843), (-38223, -43001), (-14065, 89427), (19033, -13596), (99984, -41077), (-38718, -71058), (-8164, -90642), (35191, 36100), (46665, -25325), (-99762, 63347), (-86764, -40470), (87331, 5091), (-77992, 6015), (48460, -32931), (47204, 96071), (-45810, -87292), (57715, -26007), (-52555, -9601), (25466, -17692), (88653, 23347), (-95053, -29216), (63157, 89888), (322, -89920), (86248, 57959), (20973, -59919), (47547, -38949), (-33312, 15194), (-8471, -91208), (69340, -10921), (57435, -28456), (20366, -68624), (71572, -98552), (92442, 24182), (70369, -97647), (50142, 50968), (36857, -827), (-43050, 81573), (-27400, 46492), (-35853, -74850), (20458, -97700), (-21745, -19909), (91338, 5757), (-99104, -69941), (-2642, -27979), (-51849, -39305), (46438, 94452), (32024, -14383), (95993, -35833), (-53502, -44346), (80736, -11615), (68195, 85106), (85934, -12924), (53180, -23254), (-49763, -41078), (21814, 79349), (-30345, -50047), (95180, 81013), (-7382, 67774), (-65448, 35286), (24136, -45361), (-96611, -22292), (94349, -31561), (-35761, -6538), (56572, 88629), (-71476, 25064), (60366, -31298), (-2511, 22234), (-9622, -47738), (-87180, 32282), (-42467, 37035), (-78015, 6691), (72736, 6133), (26440, 22904), (-79620, 1427), (-64017, 16278), (-85280, 5324), (-41840, -86260), (-5439, 73134), (43399, 5356), (-21691, -2626), (-40147, 9345), (46337, -60510), (80787, 5402), (-22264, 88805), (-3939, -62956), (-95895, -69181), (86569, -13557), (78676, -6887), (9900, -80614), (84804, -32575), (26798, -24834), (-93668, -2841), (41400, 98434), (56204, 6435), (46536, 46795), (41798, 48589), (34741, 40015), (-55715, 95299), (-55025, 30849), (77416, -22441), (1221, 85145), (-74288, -50432), (72334, 95037), (83589, -21768), (-11287, 14923), (-77929, -57552), (-49096, -43468), (-81758, -14523), (-22576, -23625), (65539, -92262), (83283, 79309), (51202, -46522), (47950, -96576), (14335, 32359), (59133, -98854), (81676, -65270), (95874, 60074), (60272, -11603), (28590, 94891), (-32058, 69975), (-14380, -80193), (-41659, -3005), (76231, 43233), (-72753, -34309), (59884, 8002), (-63251, -40264), (-63080, 49037), (-59865, -51239), (29751, -89202), (63671, 85926), (-31843, 15584)))
