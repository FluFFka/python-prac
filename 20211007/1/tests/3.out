((99491, 92432), (83845, 99137), (84331, 97511), (-12476, 99754), (93314, 96841), (5496, 99299), (99984, -41077))
